### PR TITLE
Updated crypto to include Rs384 and rs512

### PR DESCRIPTION
**Pull request by tiffanyachen**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Algorithms/Rs384.cs
Comment (RandalliLama) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Algorithms/Rs384.cs]: Could you make 48 a named constant?

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Algorithms/Rs512.cs
Comment (RandalliLama) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Algorithms/Rs512.cs]: The number 64 should be a constant as well.

**Comment by shahabhijeet** [comment]
@tiffanyachen can you get CLA compliant by signing in
https://cla.opensource.microsoft.com/

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Algorithms/Rs512.cs
Comment (shahabhijeet) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Algorithms/Rs512.cs]: @tiffanyachen this has to be FullNetFx (for full desktop) and !FullNetFx would be for NetCore

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Algorithms/Rs256.cs
Comment (RandalliLama) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Algorithms/Rs256.cs]: Nit: I would include 'DIGEST' in the name of the constant.  SHA256_DIGEST_LENGTH for instance.

**Review by RandalliLama — APPROVED** [review]

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Algorithms/Rs512.cs
Comment (tiffanyachen) [reply src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Algorithms/Rs512.cs]: Do you want me to change the comments?

**Comment by tiffanyachen** [comment]
@shahabhijeet I'm clicking into the link but I'm not sure how to make sure I'm CLA compliant...the page doesn't load when I hit the sign in button.

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Algorithms/Rs512.cs
Comment (schaabs) [reply src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Algorithms/Rs512.cs]: @shahabhijeet The build constants we are using here are defined in our project files, and we use these in several places throughout the KeyVault.Cryptography assembly to identify whether certain framework cryptography classes are available.  Also when I looked for the build constant FullNetFx I didn't find this at all in our branch (my best guess is this was introduced after we branched).

I'd like to merge this change as is, and then after that I will merge psSdkJson6 back into our branch, and review replacing all the build constants with the common ones at once.  Thoughts?

**Review by schaabs — APPROVED** [review]

**Comment by schaabs** [comment]
@tiffanyachen I'm closing this pull request as it needs to be resubmitted against KvDevPsSdkJson

**Pull request closed without merging**
